paper-drawer-panel.html

```diff
@@ -437,7 +437,6 @@
           this.selected = this.defaultSelected;
         }
 
-        this.setScrollDirection(this._swipeAllowed() ? 'y' : 'all');
```

Comment:
Let me know if I should do something else with this. It wasn't clear what the intention was here since this function doesn't exist.